main.go

```diff
@@ -190,9 +190,11 @@ func runObserver(dbi *gorp.DbMap, team core.AssetManagerTeam, httpListenAddr str
 
 	for _, manager := range team {
 		for _, assetType := range manager.AssetTypes() {
+			c.CloneSentryHub()
```

Comment:
This will not work the way you expect. `c` is still the same instance everywhere, so a later clone to `CloneSentryHub` would overwrite the same `c.SentryHub` that the already-spawned goroutines are using. If you want them to use separate SentryHub instances, you need to create different Context objects, i.e.

```go
func (c Context) CloneForNewGoroutine() Context {
  cloned := c
  if sendEventsToSentry {
    cloned.SentryHub = sentry.CurrentHub.Clone()
  }
}
```

and, for instance,

```go
c1 := c.CloneForNewGoroutine()
go jobLoop(func() error {
  return c1.ScrapeNextResource(...)
})
```